MetaMask/metamask-mobile
# feat: edit networks UI redesign

## **Description**
This PR adds the ability to edit networks.
This PR also includes Salim's work from the feat-edit-delete-network-menu branch.

For a short demo please view this loom: https://www.loom.com/share/1192d8d7846845e69da9894def13aeeb


## **Related issues**

Fixes:

## **Manual testing steps**

1. Click the network selector at the center top of the screen, a bottom sheet comes up with a list of networks
2. On the right side of enabled networks, a 3 dot icon is displayed as a 'more' menu, tap it to edit the network
3. The edit form shows up pre-filled with the network details to be edited

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
